src/internal/m365/collection/site/lists.go

```diff
@@ -12,21 +12,22 @@ import (
 
 // ListToSPInfo translates models.Listable metadata into searchable content
 // List Details: https://learn.microsoft.com/en-us/graph/api/resources/list?view=graph-rest-1.0
-func ListToSPInfo(lst models.Listable, size int64) *details.SharePointInfo {
+func ListToSPInfo(lst models.Listable) *details.SharePointInfo {
```

Comment:
can you please move this to `/api/lists.go`?  That's where we keep all the other `fooItemInfo` funcs.